packages/gdrcopy.spec

```diff
@@ -85,9 +129,52 @@ if ! ( /sbin/chkconfig --del gdrcopy > /dev/null 2>&1 ); then
    true
 fi              
 
+# Remove all versions from DKMS registry
+echo "Uninstalling and removing the driver."
+echo "This process may take a few minutes ..."
+dkms uninstall -m gdrdrv -v %{version} -q --all || :
+dkms remove -m gdrdrv -v %{version} -q --all --rpm_safe_upgrade || :
+
+# Clean up the weak-updates symlinks
+find /lib/modules/*/weak-updates -name "gdrdrv.ko.xz" | xargs rm
```

Comment:
is the .xz suffix present in all supported releases ? maybe a wildcard is appropriate